src/main/java/com/dynatrace/openkit/AbstractOpenKitBuilder.java

```diff
@@ -252,6 +291,10 @@ long getBeaconCacheUpperMemoryBoundary() {
         return beaconCacheUpperMemoryBoundary;
     }
 
+    DataCollectionLevel getDataCollectionLevel(){ return dataCollectionLevel; }
```

Comment:
space between `)` and `{`

Furthermore, the return can be on a separate line and `}` can also be on a separate line.